components/patient-data/impl/src/main/java/org/phenotips/data/internal/controller/RejectedGeneListController.java

```diff
@@ -41,6 +41,8 @@
 import javax.inject.Named;
 import javax.inject.Singleton;
 
+import org.apache.commons.lang.StringUtils;
```

Comment:
Wrong import, should be `org.apache.commons.lang3.StringUtils` (notice the `3` in there).